src/components/menus/content-contextual-menu.tsx

```diff
@@ -314,6 +312,7 @@ export default function ContentContextualMenu(props: Readonly<ContentContextualM
     );
 
     const [renameCB, renameState] = useDeferredFetch(
+        // @ts-ignore
```

Comment:
you can do `// @ts-expect-error refacto typing of useDefferedFetch`

Reply:
Done